packages/components/.storybook/head.html

```diff
@@ -16,4 +16,8 @@
     .tc-list-display-table tr > td + td {
         max-width: 400px;
     }
+
+    .list-container {
+        height: 80vh;
```

Comment:
This is for the list stories.
![capture d ecran de 2017-02-07 09-57-44](https://cloud.githubusercontent.com/assets/10761073/22684261/ebcbdcc0-ed1b-11e6-805b-a83c0c832b3a.png)